### PR TITLE
Topology: Apply v6 tokens

### LINKED ISSUE #133: Update CSS tokens in Topology for v6
**Goal**
Update all PatternFly variables to consume the new design tokens. This is required for the future v6 migration

### PR DESCRIPTION
**Pull request by andrew-ronaldson**
## What
Closes #133 

## Description
Updated all variables to use the new Design tokens with the v6 update.

### CODE REVIEW

**Comment by jeff-phillips-18** [comment]
@andrew-ronaldson Can you rebase so that the build failure gets resolved?

File: packages/module/src/css/topology-components.css [hunk 1]
```diff
@@ -10,241 +10,184 @@
...
-  --pf-topology__node--m-disabled--Background--Fill: var(--pf-v5-global--BackgroundColor--200);
-  --pf-topology__node--m-disabled--Background--Stroke: var(--pf-v5-global--BorderColor--100);
-  --pf-topology__node--m-info--Background--Fill: var(--pf-v5-global--primary-color--light-100);
-  --pf-topology__node--m-success--Background--Fill: var(--pf-v5-global--success-color--100);
-  --pf-topology__node--m-warning--Background--Fill: var(--pf-v5-global--warning-color--100);
-  --pf-topology__node--m-danger--Background--Fill: var(--pf-v5-global--danger-color--100);
-  --pf-topology__node--m-selected--Background--Fill: var(--pf-v5-global--active-color--100);
-
-  --pf-topology__node--m-selected--node__background--Stroke: var(--pf-v5-global--active-color--100);
-  --pf-topology__node--m-info--node__background--Stroke: var(--pf-v5-global--primary-color--light-100);
```
Comment (jeff-phillips-18) [inline packages/module/src/css/topology-components.css:29]: @andrew-ronaldson This is still used but has not been redefined

**Comment by jeff-phillips-18** [comment]
Without fixing the rest of the demo application w/ PF v6 necessary changes it's hard to tell what issues are caused by this PR vs just not setup for v6 correctly yet. However, it does look like the create connector arrows have some issues as they  the line is not correct:

Before:
image

After:
image

**Comment by jeff-phillips-18** [comment]
Surge: https://v6-tokens.surge.sh/

**Review by jenny-s51 — COMMENTED** [review]
Thank you @andrew-ronaldson for kickstarting this effort and for making these token updates!

Following discussions with you and @jeff-phillips-18 this afternoon, we decided the best way to move this effort forward is to close out this PR, pull the latest alphas / updates into v6, and apply the remaining tokens and missing styles to latest version of the demo-app as part of https://github.com/patternfly/react-topology/issues/110.

**Pull request closed without merging**